extensions/mssql/src/objectManagement/ui/objectPropertiesDialog.ts

```diff
@@ -0,0 +1,95 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as azdata from 'azdata';
+import { ObjectManagementDialogBase, ObjectManagementDialogOptions } from './objectManagementDialogBase';
+import { IObjectManagementService, ObjectManagement } from 'mssql';
+import * as localizedConstants from '../localizedConstants';
+import { AlterUserDocUrl, CreateUserDocUrl } from '../constants';
+
+export class ObjectPropertiesDialog extends ObjectManagementDialogBase<ObjectManagement.PropertiesInfo, ObjectManagement.PropertiesViewInfo> {
```

Comment:
I was thinking about the design and we can call the objectPropertiesDialog and depending on the nodeInfo it will initialize either the database properties or the server properties. I think the general tab is the only common section between the server and database properties server

Reply:
As discussed offline, let's just make a separate ServerPropertiesDialog and DatabasePropertiesDialog. There likely won't be enough overlap to warrant trying to make a common base class, and we want to avoid coupling them too tightly together since that will make future changes more difficult.